moshi/src/main/java/com/squareup/moshi/StandardJsonAdapters.java

```diff
@@ -36,27 +36,28 @@ private StandardJsonAdapters() {
     @Override public JsonAdapter<?> create(
         Type type, Set<? extends Annotation> annotations, Moshi moshi) {
       if (!annotations.isEmpty()) return null;
-      if (type == boolean.class) return BOOLEAN_JSON_ADAPTER;
-      if (type == byte.class) return BYTE_JSON_ADAPTER;
-      if (type == char.class) return CHARACTER_JSON_ADAPTER;
-      if (type == double.class) return DOUBLE_JSON_ADAPTER;
-      if (type == float.class) return FLOAT_JSON_ADAPTER;
-      if (type == int.class) return INTEGER_JSON_ADAPTER;
-      if (type == long.class) return LONG_JSON_ADAPTER;
-      if (type == short.class) return SHORT_JSON_ADAPTER;
-      if (type == Boolean.class) return BOOLEAN_JSON_ADAPTER.nullSafe();
-      if (type == Byte.class) return BYTE_JSON_ADAPTER.nullSafe();
-      if (type == Character.class) return CHARACTER_JSON_ADAPTER.nullSafe();
-      if (type == Double.class) return DOUBLE_JSON_ADAPTER.nullSafe();
-      if (type == Float.class) return FLOAT_JSON_ADAPTER.nullSafe();
-      if (type == Integer.class) return INTEGER_JSON_ADAPTER.nullSafe();
-      if (type == Long.class) return LONG_JSON_ADAPTER.nullSafe();
-      if (type == Short.class) return SHORT_JSON_ADAPTER.nullSafe();
-      if (type == String.class) return STRING_JSON_ADAPTER.nullSafe();
-      if (type == Object.class) return new ObjectJsonAdapter(moshi).nullSafe();
 
       Class<?> rawType = Types.getRawType(type);
```

Comment:
just clarifying – this part of the PR doesn’t impact behavior, does it?

Reply:
This is the core fix of the PR. It ensures `Map<String, Int>` (or whatever) in Kotlin works without having to specify `@JvmSuppressWildcards`.